README.md

```diff
@@ -53,34 +53,39 @@ which is supported by the tldr-pages project maintainers:
 
     npm install -g tldr
 
-You can also use the fully-featured [official Python client](https://github.com/tldr-pages/tldr-python-client) which can be installed via `pip3`.
+Alternatively, you can also use the fully-featured official [Python client](https://github.com/tldr-pages/tldr-python-client) which can be installed via `pip3`.
 
     pip3 install tldr
 
-That way you can write `tldr tar` in the terminal to show the tldr page for `tar`,
-just like you would write `man tar` to show its manpage.
+Then you have direct access to less complicated, easier-to-read help for commands, such as tar,
```

Comment:
Just a couple of rewording suggestions. Not a strong stance — I'm happy to discuss adjustments.

```suggestion
Then you will have direct access to simplified, easy-to-read help for commands, such as tar,
```

Reply:
I agree with the changes other than the will, let's see what others think

Reply:
No strong feelings about the "will". If you would prefer not to include it, I'd be OK with that.